storage/migrations/02_oasis_3_runtimes.up.sql

```diff
@@ -74,6 +74,32 @@ CREATE TABLE oasis_3.runtime_related_transactions
 );
 CREATE INDEX ix_runtime_related_transactions_address_height_index ON oasis_3.runtime_related_transactions (account_address, tx_round, tx_index);
 
+-- Events emitted from the runtimes. Includes deeply-parsed EVM events from EVM runtimes.
```

Comment:
This whole block is just moved from below, so that the file keeps its structure:
1) a section of tables that are (mostly) not tied to sdk-modules
2) one section per sdk module
3) permissions